### ISSUE TITLE
portus is using absolute path instead of relative path

### ISSUE THREAD

**Issue by Shuliyey** [SUSE/Portus#677]
Hi 

I deployed my portus as a docker container, and deployed another nginx container with configured proxy forward to this portus container.

everything is working great, except there're couple <a> tag link (to do with account) that's using absolute path instead of relative path. this causes problem since I'm using nginx as proxy forward.

e.g.
<img width="249" alt="2016-01-23 9 05 09" src="https://cloud.githubusercontent.com/assets/4773348/12529695/1169013e-c215-11e5-9f08-66b242d60c15.png">
the dashboard button on the left side bar has the hyper link
href="/namespaces" (relative url)
the above is fine, since it's a relative url

But the below have problem
<img width="70" alt="2016-01-23 9 04 01" src="https://cloud.githubusercontent.com/assets/4773348/12529691/eab34cde-c214-11e5-87fb-c8a460a32741.png">
the logout button has the hyper link 
http://portus/users/sign_out (absolute url)
this wouldn't work because "portus" is actually my docker container hostname, not the physical machine's domain name.

Would it be possible to change all hyper link in portus to relative path instead of using absolute_path, this would allow user to set up portus as a docker container, and have another nginx container to do a proxy forward request to the portus container.

**Comment by guiguan** [comment]
<img width="464" alt="screen shot 2016-01-24 at 2 42 07 am" src="https://cloud.githubusercontent.com/assets/940189/12531016/682fc05a-c244-11e5-949b-823eb0c989a8.png">
@Shuliyey try to change this in config.yml?

**Comment by Shuliyey** [comment]
Hi 

Sorry, after some investigation I managed to solve the problem using the below nginx configuration

```conf
http {
    .....
    server {
        .....
        location / {
	    proxy_set_header Host $http_host;
            proxy_set_header  X-Real-IP         $remote_addr; # pass on real client's IP         
            proxy_set_header  X-Forwarded-For   $proxy_add_x_forwarded_for;                      
            proxy_set_header  X-Forwarded-Proto $scheme;
```

**Comment by Shuliyey** [comment]
thx @guiguan, that helped :+1:

**Comment by mssola** [comment]
Thanks @Shuliyey for reporting this and thanks @guiguan for helping out.

> But I still think using relative path is better than using absolute path. But maybe in this case it's difficult because the link is generated using rails mvc framework

Yeah, I agree.

**Comment by mssola** [comment]
@Shuliyey @guiguan This is being addressed with #682.

**Comment by Shuliyey** [comment]
thank you @mssola  :smile:

**Comment by mssola** [comment]
@Shuliyey merged into both `master` and `v2.0`. The next patch-level release (which will be released pretty soon) will include this fix ;)
